<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>650</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1100</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QTableWidget" name="DataTable">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>600</width>
          <height>600</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="AVGLabel1">
            <property name="minimumSize">
             <size>
              <width>15</width>
              <height>20</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>15</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>第</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="AVGEdit1">
            <property name="minimumSize">
             <size>
              <width>30</width>
              <height>20</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>16777215</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="AVGLabel2">
            <property name="minimumSize">
             <size>
              <width>35</width>
              <height>20</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>35</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>列(第</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="AVGEdit2">
            <property name="minimumSize">
             <size>
              <width>30</width>
              <height>20</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>16777215</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="AVGLabel3">
            <property name="minimumSize">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>40</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>行到第</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="AVGEdit3">
            <property name="minimumSize">
             <size>
              <width>30</width>
              <height>20</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>16777215</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="AVGLabel4">
            <property name="minimumSize">
             <size>
              <width>80</width>
              <height>20</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>80</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>行）平均值为</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="AVGLabel5">
            <property name="minimumSize">
             <size>
              <width>60</width>
              <height>20</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>80</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="CalculateAVG">
          <property name="minimumSize">
           <size>
            <width>120</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>140</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>计算平均值</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="VARLabel1">
            <property name="minimumSize">
             <size>
              <width>15</width>
              <height>20</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>15</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>第</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="VAREdit1">
            <property name="minimumSize">
             <size>
              <width>30</width>
              <height>20</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>16777215</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="VARLabel2">
            <property name="minimumSize">
             <size>
              <width>35</width>
              <height>20</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>35</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>列(第</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="VAREdit2">
            <property name="minimumSize">
             <size>
              <width>30</width>
              <height>20</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>16777215</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="VARLabel3">
            <property name="minimumSize">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>40</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>行到第</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="VAREdit3">
            <property name="minimumSize">
             <size>
              <width>30</width>
              <height>20</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>16777215</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="VARLabel4">
            <property name="minimumSize">
             <size>
              <width>80</width>
              <height>20</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>80</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>行）方差为</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="VARLabel5">
            <property name="minimumSize">
             <size>
              <width>60</width>
              <height>20</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>80</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="CalculateVAR">
          <property name="minimumSize">
           <size>
            <width>120</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>140</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>计算方差</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="ColumnBar" name="ColumnBar1" native="true">
          <property name="minimumSize">
           <size>
            <width>360</width>
            <height>160</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>360</width>
            <height>160</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QPushButton" name="SaveBar1">
            <property name="minimumSize">
             <size>
              <width>60</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>80</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>保存</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="AddBar1">
            <property name="minimumSize">
             <size>
              <width>150</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>170</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>保存并加入所选列</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="UseBar1">
            <property name="minimumSize">
             <size>
              <width>150</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>170</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>保存并设为使用</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="ColumnBar" name="ColumnBar2" native="true">
          <property name="minimumSize">
           <size>
            <width>360</width>
            <height>160</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>360</width>
            <height>160</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QPushButton" name="SaveBar2">
            <property name="minimumSize">
             <size>
              <width>60</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>80</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>保存</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="AddBar2">
            <property name="minimumSize">
             <size>
              <width>150</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>170</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>保存并加入所选列</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="UseBar2">
            <property name="minimumSize">
             <size>
              <width>150</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>170</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>保存并设为使用</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1100</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>画图</string>
    </property>
    <addaction name="actionHistogram"/>
    <addaction name="actionScatterPlot"/>
    <addaction name="actionCovariance"/>
    <addaction name="actionCorrelation"/>
    <addaction name="actionPCA"/>
    <addaction name="actionKmeans"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>数据</string>
    </property>
    <addaction name="actionInput"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_3"/>
  </widget>
  <action name="actionHistogram">
   <property name="text">
    <string>直方图</string>
   </property>
  </action>
  <action name="actionScatterPlot">
   <property name="text">
    <string>散点图</string>
   </property>
  </action>
  <action name="actionCovariance">
   <property name="text">
    <string>协方差矩阵</string>
   </property>
  </action>
  <action name="actionCorrelation">
   <property name="text">
    <string>相关性矩阵</string>
   </property>
  </action>
  <action name="actionPCA">
   <property name="text">
    <string>降维绘制</string>
   </property>
  </action>
  <action name="actionInput">
   <property name="text">
    <string>导入</string>
   </property>
  </action>
  <action name="actionKmeans">
   <property name="text">
    <string>聚类分析</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ColumnBar</class>
   <extends>QWidget</extends>
   <header location="global">columnbar.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
